src/Microsoft.AspNet.Security.OpenIDConnect/NonceCache.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Security.OpenIdConnect
+{
+    public interface INonceCache
```

Comment:
This is the OIDC package. Or were you referring to System.IdentityModel.Tokens or Microsoft.IdentityModel.Protocol.Extensions?